test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Connection/ConnectionServiceTests.cs

```diff
@@ -174,23 +174,29 @@ public async Task GetCurrentConnectionStringTestwithoutApplicationName()
         /// Using connection details to build connection string
         /// </summary>
         [Test]
-        public async Task GetCurrentConnectionStringTestwithConnectionDetails()
+        public async Task GetCurrentConnectionStringTestWithConnectionDetails()
         {
             // If we make a connection to a live database 
             ConnectionService service = ConnectionService.Instance;
-            var result = LiveConnectionHelper.InitLiveConnectionInfo();
-            var resultConnectionDetails = result.ConnectionInfo.ConnectionDetails;
-            var requestContext = new Mock<SqlTools.Hosting.Protocol.RequestContext<string>>();
-
-            requestContext.Setup(x => x.SendResult(It.Is<string>((connectionString) => connectionString.Contains(resultConnectionDetails.ToString()))))
```

Comment:
the problem here was that `resultConnectionDetails.ToString()` returns the name of the type, not the connection string.  And the connection string details are for the live connection, not the connectiondetails passed to the connection service.